src/Pact/Types/Runtime.hs

```diff
@@ -215,6 +227,72 @@ newtype SPVSupport = SPVSupport {
 noSPVSupport :: SPVSupport
 noSPVSupport = SPVSupport $ \_ _ -> return $ Left $ "SPV verify not supported"
 
+-- | Confidential/Encrypted addressing info, for use in metadata on privacy-supporting platforms.
```

Comment:
let's move these to a `ChainMeta.hs` and not further clutter Runtime

Reply:
Added to `ChainMeta.hs`